### PR TITLE
added login page and signup page

### PR DESCRIPTION
**Pull request by Nagashree01**
added login page

login

added signup page
signup

https://deploy-preview-4--the-berry-store-5ed166.netlify.app/login.html
https://deploy-preview-4--the-berry-store-5ed166.netlify.app/signup.html

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
Looks good to me

**Review by kotalwar-krishna — APPROVED** [review]
Good to merge

File: login.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    display: flex;
+    justify-content: center;
+    padding: 2rem;
+    margin: 2rem;
+    
+}
+.form-div{
+    border: 1px solid var(--black-color);
+    padding: 2rem;
+    width: 35%;
```
Comment (JantuDeb) [inline login.css:11]: Use `max-width` instead so it will not shrink on mobile.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            <div class="nav-header">
+                <a href="/index.html" class="nav-title text-size-l text-center">The Berry Store</a>
+            </div>
+            <div class="search-container">
+
+                <input type="search" name="search" placeholder="search " class="search-input input-area ">
+            </div>
+            <div class="nav-right-links">
+                <button class="btn link-btn"><a class="text-style-none text-size-m" href="/login.html"
+                        target="_blank">Login</a></button>
```
Comment (JantuDeb) [inline login.html:28]: Don't put `<a>` tag inside button, link may not clicked when clicked on button.

File: login.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+            <div class="input-view">
+                <label for="password" class="label-name">Password:</label>
+                <input id="password" class="input-area" type="password" placeholder="Enter password" required />
+            </div>
+
+            <div class="sign-uplink-container">
+                <div class="checkbox-view">
+                    <input type="checkbox" class="input-checkbox">
+                    <label>Remember Me</label>
```
Comment (JantuDeb) [inline login.html:63]: You forgot to add a `for` attribute here.

File: product.html [hunk 4]
```diff
@@ -22,7 +22,7 @@
                 <input type="search" name="search" placeholder="search " class="search-input input-area ">
             </div>
             <div class="nav-right-links">
-                <button class="btn link-btn"><a class="text-style-none text-size-m" href="" target="_blank">Login</a></button>
+                <button class="btn link-btn"><a class="text-style-none text-size-m" href="/login.html" target="_blank">Login</a></button>
```
Comment (JantuDeb) [inline product.html:25]: Same here, you can apply the styles of button to `<a> ` tag. or wrap button with `<a>`

**Review by JantuDeb — APPROVED** [review]
Overall code looks nice and.structured. I also added some comments which can be improved.

File: login.html (hunk 2 above)
Comment (Nagashree01) [reply login.html]: yes I'll do this. Did not think of this. Thank you!

**Pull request merged**